frontend/src/bundles/common/components/input/input.enum.ts

```diff
@@ -0,0 +1,11 @@
+enum type {
+    EMAIL = 'email',
+    TEXT = 'text',
+    PASSWORD = 'password',
+}
+
+enum placeholder {
+    BULLET = '\u2022',
+}
+
+export { placeholder, type };
```

Comment:
This enum seems unnecessary since it's not being used. 
The placeholder should be a constant. 
For type='password', simply use six bullets as a placeholder without passing it via props.
Also fix file structure